mifospay/src/main/java/org/mifospay/bank/setupUpi/screens/OtpScreen.kt

```diff
@@ -0,0 +1,90 @@
+package org.mifos.mobilewallet.mifospay.seup_upi.fragment
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mifos.mobilewallet.mifospay.ui.VerifyStepHeader
+import org.mifospay.core.designsystem.theme.MifosTheme
+import org.mifospay.R
+import org.mifos.mobilewallet.mifospay.ui.OtpTextField
+
+
+@Composable
+fun OtpScreen(
+    verificationStatus: Boolean = false,
+    contentVisibility: Boolean = false,
+    realOtp: String,
+    onOtpTextCorrectlyEntered: () -> Unit
+) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(
+                top = dimensionResource(id = R.dimen.value_15dp),
+                bottom = dimensionResource(id = R.dimen.value_15dp),
+                start = dimensionResource(id = R.dimen.value_10dp),
+                end = dimensionResource(id = R.dimen.value_10dp)
+            ), elevation = 1.dp
+    ) {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            VerifyStepHeader("OtpSetUp", verificationStatus)
+            if (contentVisibility) {
+                OtpScreenContent(realOtp, onOtpTextCorrectlyEntered)
+            }
+        }
+    }
+}
+
+@Composable
+private fun OtpScreenContent(realOtp: String, onOtpTextCorrectlyEntered: () -> Unit) {
+    Text(
+        text = stringResource(id = R.string.enter_otp),
+        color = Color(0xFF1E90FF),
+        fontSize = 18.sp,
+        style = MaterialTheme.typography.headlineMedium
```

Comment:
move to textstyle and also make sure that the color resource is defined in Color.kt file